src/main/java/org/jboss/pnc/api/deliverablesanalyzer/dto/AnalyzePayload.java

```diff
@@ -32,6 +32,11 @@
 @Jacksonized
 @Builder
 public class AnalyzePayload implements Serializable {
+
+    private static final long serialVersionUID = -2043312712105992478L;
+
+    private String operationId;
```

Comment:
Does the analysis service need the `operationId`? If it's needed for a reference only, we should put in the callback. If it's for logging mdc only, it should be in the request headers.

Reply:
The operationId (see https://github.com/project-ncl/deliverables-analyzer/pull/243/files) would be used in the heartbeat and the DelAnalyzer statuses cache. I found it odd to rely on a sha of the first url among the urls payload, using the same operationId. If we exposed the content of the cache, we could monitor easily where in the process is a particular operation.